lutris/game.py

```diff
@@ -403,6 +420,9 @@ def on_game_quit(self):
            or self.runner.system_config.get('reset_desktop'):
             display.change_resolution(self.original_outputs)
 
+        if self.compositor_disabled:
+            subprocess.Popen(self.start_compositor, shell=True)
```

Comment:
use `lutris.system.execute` instead of Popen calls

Reply:
Done. I had to allow system.execute to optionally run the command in a shell (default is still False, so it shouldn't break compatibility).